kubernetes/bosh-cloud-config.yml

```diff
@@ -0,0 +1,17 @@
+---
+- type: replace
+  path: /vm_extensions?/name=bosh/cloud_properties?
+  value:
+    services:
+      - name: inner-bosh
+        type: NodePort
+        ports:
+          - name: nats
+            port: 4222
+            nodePort: ((port-nats))
```

Comment:
general convention within this project: all variables should have underscores instead of dashes.